cmd/ci-secret-bootstrap/main_test.go

```diff
@@ -280,39 +244,37 @@ var (
 		BearerToken: "token2",
 	}
 
+	defaultConfigFrom = map[string]secretbootstrap.BitWardenContext{
```

Comment:
Can you revert moving this please? Makes it harder to review and makes the tests share state